modules/siddhi-core/src/main/java/org/wso2/siddhi/core/executor/function/InstanceOfBooleanFunctionExecutor.java

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
Thanks for pointing it out @prabathabey, added the changed in pull request - https://github.com/wso2/siddhi/pull/74